### PR TITLE
[PM-7562] Add DuckDuckGo back to State Service

### PR DESCRIPTION
**Pull request by differsthecat**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Add the previous enable DDG setting back so that users on previous versions of DuckDuckGo can use the integration.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/desktop/src/app/accounts/settings.component.ts [hunk 1]
```diff
@@ -268,9 +268,10 @@ export class SettingsComponent implements OnInit {
       enableBrowserIntegration: await this.stateService.getEnableBrowserIntegration(),
       enableBrowserIntegrationFingerprint:
         await this.stateService.getEnableBrowserIntegrationFingerprint(),
-      enableDuckDuckGoBrowserIntegration: await firstValueFrom(
-        this.desktopAutofillSettingsService.enableDuckDuckGoBrowserIntegration$,
-      ),
+      enableDuckDuckGoBrowserIntegration:
```
Comment (MGibson1) [inline apps/desktop/src/app/accounts/settings.component.ts:271]: I think we'll want to choose one of these for the read source and just make sure we write to both

**Review by MGibson1 — CHANGES_REQUESTED** [review]
If I understand the situation correctly, we just need to ensure the value is stored to the same location for DDG to read. We don't need to use it as a source of data for _ourselves_, correct?

In that case, we should only add the `set` method back and maintain the state provider location for our own value until DDG can update.

File: apps/desktop/src/app/accounts/settings.component.ts (hunk 1 above)
Comment (differsthecat) [reply apps/desktop/src/app/accounts/settings.component.ts]: Makes total sense, we don't need to read it, DDG does so no need for the getter

Comment (differsthecat) [reply apps/desktop/src/app/accounts/settings.component.ts]: Fixed!

**Review by MGibson1 — APPROVED** [review]

**Pull request merged**